getting_started.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Getting Started"
+nav_order: 2
+description: >
+    Getting Started with FalkorDB Graph Database.
+---
+
+
+# Getting Started with FalkorDB
+
+This guide will walk you through setting up FalkorDB, modeling a social network as a graph, 
+and accessing it using the [FalkorDB Python client](/clients) with [Cypher](/cypher).
+
+---
+
+## Prerequisites
+
+1. **FalkorDB Instance**: Set up FalkorDB (on-prem or cloud). [Learn More](/configuration).  
+2. **Python Installed**: Ensure you have Python 3.8+ installed.  
+3. **Install FalkorDB Python Client**:
+   
+   ```bash
+   pip install falkordb
+   ```
+   
+---
+
+## Step 1: Model a Social Network as a Graph
+
+Let's create a simple graph for a social network where:  
+- **Nodes** represent `User` and `Post`.
+- **Relationships** represent `FRIENDS_WITH` and `CREATED`.
```

Comment:
- **Relationships** connect `Users` with a `FRIENDS_WITH` relation, and `Users` are connected via a `CREATED` relations to `Posts`